pyproject.toml

```diff
@@ -1,29 +1,23 @@
-[build-system]
-requires = ["flit_core >=3.2,<4"]
-build-backend = "flit_core.buildapi"
-
-[project]
+[tool.poetry]
 name = "tawazi"
-authors = [{name = "Bashir ABDEL WAHED", email = "bashir@mindee.co"}]
+version = "0.1.2"
+description = "This library helps you execute a set of functions in a Directed Acyclic Graph (DAG) dependency structure in parallel. It aims at providing This in a production environment"
```

Comment:
There is a mistake here: 
I suggest something along the lines of:
```
...(DAG) dependency structure in parallel in a production environment.
```

Reply:
Yep you're right I forgot something, will change now